src/renderer/components/ScreenSharePicker.tsx

```diff
@@ -453,6 +453,25 @@ function StreamSettings({
                     />
                 )}
             </Card>
+
+            <div className="vcd-screen-picker-save-settings">
+                <label htmlFor="" className="vcd-screen-picker-save-label">
+                    Remember stream settings
+                </label>
+                <div className="vcd-screen-picker-save-control">
+                    <div
+                        className={`vcd-screen-picker-save-checkbox ${Settings.stream?.preferred && "checked"}`}
+                        onClick={() => {
```

Comment:
this markup is very messy and not semantic (using a plain div as a checkbox is very bad. at the very least it should have aria attributes like `role` that tell user agents that this is a checkbox for visually impaired people and other users of screenreaders). have a look at the rest of Vesktop (& Vencord) Settings UI and see how those implement things like Switches, Checkboxes. Vencord exports a bunch of Discord Components you can use instead of implementing stuff yourself